README.md

```diff
@@ -50,6 +40,8 @@ If you want to integrate StreamChat SDKs into your app, you can already use vers
 * [StreamChat SDK (low-level)](https://github.com/GetStream/stream-chat-swift/tree/main/Sources/StreamChat): An SDK to use if you want to build fully-custom UI. `StreamChatUI` uses `StreamChat` for communicating with Stream chat backend.
 * [SDK Documentation](https://github.com/GetStream/stream-chat-swift/wiki)
 
+**Information about the last stable version can be found at [v2.x README](https://github.com/GetStream/stream-chat-swift/tree/main_v2/).**
+
```

Comment:
The last stable version is `3.1` ;-) This should probably be something like: Information about the version 2.x of the framework...